eva-accession-ws/src/test/java/uk/ac/ebi/eva/accession/ws/SubmittedVariantsRestControllerTest.java

```diff
@@ -503,4 +504,25 @@ public void testGetDeprecatedDbsnpSubmittedVariant()
         assertEquals(variant1, response.getBody().get(0).getData());
         assertCreatedDateNotNull(response.getBody());
     }
+
+    @Test
+    public void testGetNonExistentDbsnpSubmittedVariant()
+            throws AccessionMergedException, AccessionDoesNotExistException, AccessionDeprecatedException {
+        List<Long> generatedAccessionNumbers =
+                generatedAccessions.stream().map(AccessionWrapper::getAccession).collect(Collectors.toList());
+        Long minAccession = Collections.min(generatedAccessionNumbers);
+        Long maxAccession = Collections.max(generatedAccessionNumbers);
+
+        String getVariantUrl = URL + ("" + (maxAccession + 1));
```

Comment:
`URL + (maxAccession + 1)` works, and if you want to make the cast explicit, I'd prefer the below code rather than concatenating an empty string.

```suggestion
        String getVariantUrl = URL + Long.toString(maxAccession + 1);
```